shared/setup.cfg

```
@@ -0,0 +1,21 @@
+[bdist_wheel]
+universal = 1
+
+[flake8]
+exclude = docs
+ignore = W503,E302,E203,F841
+max-line-length = 88
+
+[aliases]
+
+[tool:isort]
+line_length = 88
+force_grid_wrap = 0
+include_trailing_comma = true
+multi_line_output = 3
+use_parentheses = true
+lines_after_imports = 2
+default_section = THIRDPARTY
+sections = FUTURE,STDLIB,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
+known_first_party = fv3core,fv3gfs,pace
+known_third_party = f90nml,pytest,xarray,numpy,mpi4py,gt4py
```

Because isort and flake8 are run by the top-level pre-commit, you shouldn't need to copy its configuration between subprojects. If I'm wrong and this breaks the tool, please let me know.

```suggestion
[aliases]

```